samples/DistributedWebCrawler/DistributedWebCrawler.fs

```diff
@@ -128,6 +127,8 @@ let main orgargs =
         else
             Cluster.StartCluster( PrajnaClusterFile )
         let cluster = Cluster.GetCurrent()
+        let remote = DSet<_>(Name = "remoteExec")
+        remote.Execute(fun () -> ())
```

Comment:
To attach debugger at node. It basicaly just launches container.


Sent via the Samsung GALAXY S® 5, an AT&T 4G LTE smartphone

-------- Original message --------
From: Weirong Zhu <notifications@github.com>
Date: 2015/12/08  19:04  (GMT-08:00)
To: MSRCCS/Prajna <Prajna@noreply.github.com>
Cc: sanjeevm0 <sanjeevm@stanfordalumni.org>
Subject: Re: [Prajna] Fix issue caused by early disposal of MemoryStreamB (#116)

> @@ -128,6 +127,8 @@ let main orgargs =
>          else
>              Cluster.StartCluster( PrajnaClusterFile )
>          let cluster = Cluster.GetCurrent()
> +        let remote = DSet<_>(Name = "remoteExec")
> +        remote.Execute(fun () -> ())

What's the purpose of the new code here? For debugging?

---
Reply to this email directly or view it on GitHub:
https://github.com/MSRCCS/Prajna/pull/116/files#r47046559